apps/aehttp/test/aehttp_integration_SUITE.erl

```diff
@@ -1581,29 +1588,39 @@ acc_txs_test(Pubkey, Offset, Limit) ->
                         end,
-                    {ok, AllPendingTxs} = rpc(aec_tx_pool, peek, [infinity]),
-                    FilteredPendingTxs =
-                        lists:filter(
-                            fun(Tx) ->
-                                PubkeyInTx(Tx) andalso
-                                apply_tx_type_filter(TT, [Tx]) =/= []
-                            end,
-                            AllPendingTxs),
-                    AccountPendingTx = encode_pending_tx(
-                                         lists:sort(fun(A, B) ->
-                                                        aetx:nonce(aetx_sign:tx(A)) >=
-                                                        aetx:nonce(aetx_sign:tx(B))
-                                                    end,
-                                                    FilteredPendingTxs),
-                                        TxEncoding),
+                    AccountPendingTx =
+                        case ShowPending of
+                            false -> [];
+                            _ ->
+                                {ok, AllPendingTxs} = rpc(aec_tx_pool, peek, [infinity]),
+                                FilteredPendingTxs =
+                                    lists:filter(
+                                        fun(Tx) ->
+                                            PubkeyInTx(Tx) andalso
+                                            apply_tx_type_filter(TT, [Tx]) =/= []
+                                        end,
+                                        AllPendingTxs),
+                                encode_pending_tx(
+                                    lists:sort(fun(A, B) ->
+                                                    aetx:nonce(aetx_sign:tx(A)) >=
+                                                    aetx:nonce(aetx_sign:tx(B))
+                                                end,
+                                                FilteredPendingTxs),
+                                    TxEncoding)
+                        end,
                     #{<<"data_schema">> := ExpectedDS,
                       <<"transactions">> := AllTxs} =
                         expected_range_result(0, ToHeight, TxEncoding, TT,
                                               PubkeyInTx, true),
-                    Params = maps:merge(
-                               maps:merge(tx_encoding_param(TxEncoding),
-                                          make_tx_types_filter(Filter)),
-                               #{offset => Offset, limit => Limit}),
+                    ParamsL0 = [tx_encoding_param(TxEncoding),
+                                make_tx_types_filter(Filter),
+                                #{offset => Offset, limit => Limit}],
+                    ParamsL =
+                        case ShowPending of
+                            default -> ParamsL0;
+                            _ -> [{pending, atom_to_binary(ShowPending, utf8)} | ParamsL0]
```

Comment:
Please no open clause here - it makes test more fragile and less readable (I see clue is `[default, true, false]` in caller function).